python/nx-cugraph/nx_cugraph/interface.py

```diff
@@ -246,6 +246,39 @@ def key(testpath):
             key("test_tree_isomorphism.py:test_positive"): too_slow,
             key("test_tree_isomorphism.py:test_negative"): too_slow,
             key("test_efficiency.py:TestEfficiency.test_using_ego_graph"): maybe_oom,
+            # These repeatedly call `bfs_layers`, which converts the graph every call
```

Comment:
Is this reason enough to move this PR to Draft until there's some kind of caching mechanism added?

Reply:
I don't think that's necessary. I think this PR is reviewable and mergable. We already have this of repeated conversions elsewhere. Perhaps we could create an issue to track this.

Reply:
ok, I created this issue: https://github.com/rapidsai/cugraph/issues/4055